clients/vscode-hlasmplugin/src/test/syntaxes/endevorLong/basics.listing.snap

```diff
@@ -287,20 +286,18 @@
 #                                                                 ^ source.hlasmListing
 #                                                                  ^^^ source.hlasmListing punctuation.definition.tag.hlasm
 #                                                                     ^^^^ source.hlasmListing
-#                                                                         ^^^^^^^ source.hlasmListing comment.hlasm
-#                                                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing
+#                                                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing comment.hlasm
```

Comment:
Again, spaces before `REMARKS` (except the one right after `OP1,OP2`) should be part of the comment.